app/views/submissions/show/_status.html.haml

```diff
@@ -23,7 +23,9 @@
       - else
         Submitted on #{l submission.updated_at, format: :short}
 
-        = submission.decorate.doi
+        - if submission.published?
```

Comment:
Theoretically this is always true if DOI is not nil, but perhaps you are right - it is good to double check things.